tests/k6chai.js

```diff
@@ -2,6 +2,7 @@ import { expect as expect4340, describe as describe4340 } from '../lib/k6chaijs/
 import { expect as expect4341, describe as describe4341 } from '../lib/k6chaijs/4.3.4.1/index.js'
 import { expect as expect4342, describe as describe4342 } from '../lib/k6chaijs/4.3.4.2/index.js'
 import { expect as expect4343, describe as describe4343 } from '../lib/k6chaijs/4.3.4.3/index.js'
+import { expect as expect4501, describe as describe4501 } from '../lib/k6chaijs/4.5.0.1/index.js'
```

Comment:
I see these changes are missing for `v4.5.0.0`, so perhaps this is a good opportunity to add them? 🤔

Reply:
Fixed in [32c36c2](https://github.com/grafana/jslib.k6.io/pull/139/commits/32c36c2f5a91bb10817d9d77eba490d62ea79335).

Reply:
Great, thanks! :)